<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#150050;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton#SeeDetailsButton {
background-color:#FF4C29;
color:white;
}

QPushButton#SeeDetailsButton:hover {
    background-color:white;
color:#FF4C29;
}

QPushButton#SeeDetailsButton:pressed {
    background-color: rgb(224, 0, 0);     
}
QPushButton#AddTransactionButton {
background-color:#FF4C29;
color:white;
}

QPushButton#AddTransactionButton:hover {
    background-color:white;
color:#FF4C29;
}

QPushButton#AddTransactionButton:pressed {
    background-color: rgb(224, 0, 0);     
}
QPushButton#BackButton {
background-color:#FF4C29;
color:white;
}

QPushButton#BackButton:hover {
    background-color:white;
color:#FF4C29;
}

QPushButton#BackButton:pressed {
    background-color: rgb(224, 0, 0);     
}</string>
   </property>
   <widget class="QPushButton" name="SeeDetailsButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>400</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>See Details</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>390</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FF4C29;</string>
    </property>
    <property name="text">
     <string>Enter Student Id</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idlineEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>400</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1px solid white;
border-radius : 10px;
color : white;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>500</y>
      <width>700</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white;</string>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>200</number>
    </attribute>
    <column>
     <property name="text">
      <string>Amount</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>700</y>
      <width>251</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FF4C29;</string>
    </property>
    <property name="title">
     <string>Total Fees</string>
    </property>
    <widget class="QLabel" name="totalAmountLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>700</y>
      <width>261</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FF4C29;</string>
    </property>
    <property name="title">
     <string>Fess Paid</string>
    </property>
    <widget class="QLabel" name="feesPaidLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>700</y>
      <width>241</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FF4C29;</string>
    </property>
    <property name="title">
     <string>Fees Remaining</string>
    </property>
    <widget class="QLabel" name="feesRemainingLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>911</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FF4C29;</string>
    </property>
    <property name="title">
     <string>Add Transaction</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#FF4C29;</string>
     </property>
     <property name="text">
      <string>Transaction Amount</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="descriptionLineEdit">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>170</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 1px solid white;
border-radius : 10px;
color : white;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>70</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 1px solid white;
border-radius : 10px;
color : white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#FF4C29;</string>
     </property>
     <property name="text">
      <string>Date of Transaction</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>160</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#FF4C29;</string>
     </property>
     <property name="text">
      <string>Transaction Description</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="amountLineEdit">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>120</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 1px solid white;
border-radius : 10px;
color : white;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="error">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>210</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#FF4C29;</string>
     </property>
     <property name="text">
      <string>Enter Student Id</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="idlineEdit_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>220</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 1px solid white;
border-radius : 10px;
color : white;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Error">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>450</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FF4C29;</string>
    </property>
    <property name="text">
     <string>TRANSACTION HISTORY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddTransactionButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>350</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Transaction</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FF4C29;</string>
    </property>
    <property name="text">
     <string>FEES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="error1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
